tests/packagedcode/data/pypi/unpacked_sdist/prefer-egg-info-pkg-info/celery-expected.json

```
@@ -0,0 +1,8105 @@
+      "extra_data": {},
+      "dependency_uid": "pkg:pypi/boto3?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "for_package_uid": "pkg:pypi/celery@5.2.7?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "datafile_path": "celery/celery.egg-info/PKG-INFO",
+      "datasource_id": "pypi_editable_egg_pkginfo"
+    },
+    {
+      "purl": "pkg:pypi/elasticsearch",
+      "extracted_requirement": "elasticsearch; extra == \"elasticsearch\"",
+      "scope": "elasticsearch",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false,
+      "resolved_package": {},
+      "extra_data": {},
+      "dependency_uid": "pkg:pypi/elasticsearch?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "for_package_uid": "pkg:pypi/celery@5.2.7?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "datafile_path": "celery/celery.egg-info/PKG-INFO",
+      "datasource_id": "pypi_editable_egg_pkginfo"
+    },
+    {
+      "purl": "pkg:pypi/eventlet",
+      "extracted_requirement": "eventlet>=0.32.0; python_version < \"3.10\" and extra == \"eventlet\"",
+      "scope": "eventlet",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false,
+      "resolved_package": {},
+      "extra_data": {},
+      "dependency_uid": "pkg:pypi/eventlet?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "for_package_uid": "pkg:pypi/celery@5.2.7?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "datafile_path": "celery/celery.egg-info/PKG-INFO",
+      "datasource_id": "pypi_editable_egg_pkginfo"
+    },
+    {
+      "purl": "pkg:pypi/gevent",
+      "extracted_requirement": "gevent>=1.5.0; extra == \"gevent\"",
+      "scope": "gevent",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false,
+      "resolved_package": {},
+      "extra_data": {},
+      "dependency_uid": "pkg:pypi/gevent?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "for_package_uid": "pkg:pypi/celery@5.2.7?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "datafile_path": "celery/celery.egg-info/PKG-INFO",
+      "datasource_id": "pypi_editable_egg_pkginfo"
+    },
+    {
+      "purl": "pkg:pypi/librabbitmq",
+      "extracted_requirement": "librabbitmq>=1.5.0; extra == \"librabbitmq\"",
+      "scope": "librabbitmq",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false,
+      "resolved_package": {},
+      "extra_data": {},
+      "dependency_uid": "pkg:pypi/librabbitmq?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "for_package_uid": "pkg:pypi/celery@5.2.7?uuid=fixed-uid-done-for-testing-5642512d1758",
+      "datafile_path": "celery/celery.egg-info/PKG-INFO",
+      "datasource_id": "pypi_editable_egg_pkginfo"
+    },
+    {
+      "purl": "pkg:pypi/pylibmc",
+      "extracted_requirement": "pylibmc; platform_system != \"Windows\" and extra == \"memcache\"",
+      "scope": "memcache",
+      "is_runtime": true,
+      "is_optional": false,
```

IMHO we should be true for "extras" e.g. not an "install_requires" and therefore not in "install" scope:
```suggestion
      "is_optional": true,
```